$NetBSD: patch-ac,v 1.3 2007/01/06 19:06:22 rillig Exp $

qt3-3.3.5 fix. Can probably be removed after next qt3 release.

--- src/mypasswordsafe.ui.orig	2004-12-06 17:03:46.000000000 +0100
+++ src/mypasswordsafe.ui
@@ -1006,6 +1006,5 @@
 <pixmapinproject/>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>safelistview.hpp</includehint>
 </includehints>
 </UI>
